dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/sql.vue

```diff
@@ -43,6 +43,13 @@
         </div>
       </div>
     </m-list-box>
+    <m-list-box>
+        <div slot="text"><strong class='requiredIcon'>*</strong>{{$t('Max Numbers Return')}}</div>
+        <div slot="content">
+          <el-input type="input" :disabled="isDetails" size="medium" v-model="limit" :placeholder="$t('Max Numbers Return placeholder')">
+          </el-input>
+        </div>
+    </m-list-box>
```

Comment:
This limit number of rows only needs to be displayed in the query scenario, and it is recommended that the backend also set the initial value of limit at the same time

Reply:
@zhuangchong Thanks for youre review.
Sorry I forget to merge some code. I have updated by add a condition `<m-list-box v-show="sqlType === '0'">`